### DISCUSSION TITLE
米忽悠重拳出击，看来API是不会改了

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#505] [General]
https://bbs.nga.cn/read.php?tid=28002663

![image](https://user-images.githubusercontent.com/9006264/146631298-6325c0d2-2e42-4c9a-8d27-bf55e0cde082.png)

![image](https://user-images.githubusercontent.com/9006264/146631296-7611b132-5d50-4baa-90d5-08200e41daf2.jpeg)

**Comment by Arondight** [comment]
这只是一个dmca，而且很可能还没啥用，参考内鬼网正面刚赢了米忽悠还删了两巴掌，不过我收到了会跑路，因为我所有的社交软件都在实名上网，容易影响我工作

**Reply by Arondight** [reply]
@SilveryStar 老大哥你得关注一下同类项目，你一个学生被搞一下也麻烦，虽然我觉得问题不大，不过比较影响学业

**Comment by mark9804** [comment]
我私信去问了发帖的大佬被投诉的项目的具体功能，希望有回复吧，我认为用了游戏素材只能算是个万能理由，实际还是mhy觉得打扰到他们赚钱了才开拳

**Comment by mark9804** [comment]
这波狙的是抽卡分析小程序
![image](https://user-images.githubusercontent.com/9006264/147365397-a935bbd5-fe63-4bfa-bcb2-d3dc68f796dc.jpeg)![image](https://user-images.githubusercontent.com/9006264/147365405-6e574a3e-48e4-454b-bf69-76a6eaceb005.jpeg)

**Comment by wangan2021** [comment]
三年封闭、只进不出，明抢？
欲要提现，必先拉人，离谱！
